modules/serverless-build-func-kn.adoc

```diff
@@ -8,6 +8,26 @@
 
 Before you can run a function, you must build the function project by using the `kn func build` command. This command creates an OCI container image that can be run locally on your computer or on an {product-title} cluster. The build command uses the function project name and the image registry name to construct a fully qualified image name for your function.
 
+By default, `kn func build` creates a container image using the link:https://buildpacks.io/[CNCF Cloud Native Buildpacks] technology. To use the Red Hat Source-to-Image (S2I) technology instead, supply the `-b` flag:
```

Comment:
buildpack is the default for the upstream func, so I think it is better invert this sentence

Reply:
@jrangelramos The discussions in https://issues.redhat.com/browse/SRVOCF-422 (last comment) and in https://issues.redhat.com/browse/SRVOCF-418 (several comments) suggest otherwise.
@lance For downstream buildpack is the default too for 1.24, right?

Reply:
I know it's confusing, but 1.24 uses `s2i` as the default in the downstream and `pack` as the default in all other releases. I was confused myself when I lgtm this PR. Sorry for the mixup! Here is the relevant default setting for 1.24 https://github.com/openshift-knative/kn-plugin-func/blob/2cdb44f832ff156f4aa7a384ff315eccea394693/cmd/build.go#L53

This is reverted in 1.25. I think Naina was off-by-one in her Jira comment (comp sci's most difficult problem!)

Reply:
yea.. here is the patch that makes s2i default on midstream https://github.com/openshift-knative/kn-plugin-func/blob/main/openshift/patches/0002-default-build.patch

Reply:
Thanks @jrangelramos , well-spotted. Thanks @lance for clarification.

Please give this one last glance at https://github.com/openshift/openshift-docs/pull/48899/commits/d70e19fa7421d2a691b7e3ca60a63f106bc7bd07 (I swapped `s2i` and `pack` for 1.24 and I removed the note about 1.23 because it's now clear that 1.23 and 1.24 have the same defaults).